File: src/main/scala/com/vc/easy/L168.scala
package com.vc.easy

object L168 {
  def convertToTitle(n: Int): String = {
    import scala.collection.mutable
    val st = new mutable.Stack[Char]()
    var nVar = n
    while(nVar > 0) {
      val ch = ((nVar - 1) % 26 + 'A').toChar
      nVar = (nVar - 1) / 26
      st.push(ch)
    }
    st.mkString("")
  }
}

File: src/main/java/com/vc/easy/L168.java
package com.vc.easy;

class L168 {
    public String convertToTitle(int n) {
        StringBuilder sb = new StringBuilder();
        while(n > 0) {
            char ch = (char)((n - 1) % 26 + 'A');
            n = (n - 1) / 26;
            sb.insert(0, ch);
        }
        return sb.toString();
    }
}
